archunit/src/main/java/com/tngtech/archunit/core/domain/JavaModifier.java

```diff
@@ -46,7 +46,11 @@
     @PublicAPI(usage = ACCESS)
     SYNCHRONIZED(EnumSet.of(ApplicableType.METHOD), Opcodes.ACC_SYNCHRONIZED),
     @PublicAPI(usage = ACCESS)
-    NATIVE(EnumSet.of(ApplicableType.METHOD), Opcodes.ACC_NATIVE);
+    NATIVE(EnumSet.of(ApplicableType.METHOD), Opcodes.ACC_NATIVE),
+    @PublicAPI(usage = ACCESS)
```

Comment:
not sure about bridge (they are filteredout), but synthetic ctors are visible for archunit